### DISCUSSION TITLE
Which ARC is this?

### DISCUSSION THREAD

**Discussion by shankarRaman** [actions/actions-runner-controller#3218] [General]
Saw this post https://github.blog/changelog/2024-01-10-reduce-job-queue-times-with-newly-updated-min-attribute-in-action-runner-controller/ and was curious if this is the legacy ARC or the GitHub supported ARC

**Comment by shankarRaman** [comment]
@nikola-jokic @Link- please let me know which ARC is this. Perhaps there's a better way for me identify this and if so please do let me know :-)

**Reply by Link-** [reply]
Everything that is announced in the GitHub changelog is related to the runner scale set modes. The GitHub supported version of ARC. https://github.com/actions/actions-runner-controller/discussions/2775

**Reply by shankarRaman** [reply]
Awesome, thanks @Link-
